-1

**Failed Tests**:   Build  HeaderConsistency 
**Size**: This PR adds an extra 64KB to repository
**Summary**: https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-8cc1fe/47928/summary.html
**COMMIT**: 735aba2b825052c9bcf3347e90e0355e0340ddd7
**CMSSW**: CMSSW_15_1_X_2025-09-01-1100/el8_amd64_gcc12
**User test area**: For local testing, you can use `/cvmfs/cms-ci.cern.ch/week1/cms-sw/cmssw/48826/47928/install.sh` to create a dev area with all the needed externals and cmssw changes.


## Build

I found compilation error when building: 

<pre>/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/gcc/12.3.1-40d504be6370b5a30e3947a6e575ca28/bin/c++ -c -DCMS_MICRO_ARCH='x86-64-v3' -DGNU_GCC -D_GNU_SOURCE -DTBB_USE_GLIBCXX_VERSION=120301 -DTBB_SUPPRESS_DEPRECATED_MESSAGES -DTBB_PREVIEW_RESUMABLE_TASKS=1 -DTBB_PREVIEW_TASK_GROUP_EXTENSIONS=1 -DBOOST_SPIRIT_THREADSAFE -DPHOENIX_THREADSAFE -DBOOST_MATH_DISABLE_STD_FPCLASSIFY -DBOOST_UUID_RANDOM_PROVIDER_FORCE_POSIX -DBOOST_MPL_IGNORE_PARENTHESES_WARNING -DGNU_GCC -DG4V9 -DGNU_GCC -DG4V9 -DDD4HEP_USE_GEANT4_UNITS=1 -DCMSSW_GIT_HASH='CMSSW_15_1_X_2025-09-01-1100' -DPROJECT_NAME='CMSSW' -DPROJECT_VERSION='CMSSW_15_1_X_2025-09-01-1100' -Isrc -Ipoison -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/cms/cmssw-patch/CMSSW_15_1_X_2025-09-01-1100/src -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/cms/coral/CORAL_2_3_21-5febecd7b3082c7b62e92719f54f5ed2/include/LCG -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/dd4hep/v01-29-00-20154c7d2051779000c69dd6f1e43dab/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/alpaka/1.2.0-ab54b9441c1c5ea12d3b50618a416376/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/geant4/11.2.2-7ace3faa1f613dee2af01c298c5e704c/include/Geant4 -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/geant4/11.2.2-7ace3faa1f613dee2af01c298c5e704c/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/libpng/1.6.37-04d5aac74fb0e0b044d856194f5a4a98/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/pcre/8.43-2d141998cfe5424b8f7aff48035cc2da/include -isystem/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/boost/1.80.0-9fe8bbbc2e1ad17df3cd25532722ba62/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/bz2lib/1.0.6-d065ccd79984efc6d4660f410e4c81de/include -isystem/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/clhep/2.4.7.1-d3a3e353d370e701238f7949a0d7909f/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/curl/7.79.0-d8c0d5017cfad573d276c56bd08114b8/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/expat/2.4.8-b093687a482bf386f8f8c236c5b2efa2/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/giflib/5.2.1-c9b71bd305c04482627fc5451da9661d/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/gsl/2.6-f7574c606b0ce57ff601d3ca9534cd01/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/heppdt/3.04.01-ed47955de25516cfc664d37543c8fc1d/include -isystem/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/hls/2025.05-4da036171b10090d7d21e15c49bb27a1/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/libjpeg-turbo/2.0.2-a09ac163eb10711ddf15e845fb1234c1/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/libuuid/2.34-27ce4c3579b5b1de2808ea9c4cd8ed29/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/protobuf/3.21.9-1126508a53768c90e66f6bf1821ac03a/include -isystem/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/lcg/root/6.32.13-be42ecc3174fc32011283ebcc8be2a76/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/sqlite/3.36.0-6a61dda6b4338fb9b6c26060efd5fd9a/include -isystem/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/tbb/v2022.0.0-d2baf29793e68d7b624be49d552519ec/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/cms/vdt/0.4.3-f008bd09702a81e8a062bbef698ed3af/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/vecgeom/v1.2.11-db93479c7d2200c4de9dccc28799f670/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/vecgeom/v1.2.11-db93479c7d2200c4de9dccc28799f670/include/VecGeom -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/xerces-c/3.1.3-c7b88eaa36d0408120f3c29826a04bf6/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/xz/5.2.5-6f3f49b07db84e10c9be594a1176c114/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/zlib/1.2.13-d217cdbdd8d586e845e05946de2796be/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/cuda/12.9.0-cd4613335deff6e5576167f7dbef4cb0/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/eigen/3bb6a48d8c171cf20b5f8e48bfb4e424fbd4f79e-5d91c922e771c0dc4f6bc00f61f3e2c5/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/eigen/3bb6a48d8c171cf20b5f8e48bfb4e424fbd4f79e-5d91c922e771c0dc4f6bc00f61f3e2c5/include/eigen3 -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/fmt/10.2.1-e35fd1db5eb3abc8ac0452e8ee427196/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/json/3.11.3-fe8b14080d05d6ff46ded7c91e3977bc/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/md5/1.0.0-5b594b264e04ae51e893b1d69a797ec6/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/OpenBLAS/0.3.27-70a9dd2c9f309171934f13e3003b0540/include -isystem/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/tensorflow/2.12.0-d8b7f62cac965299506ba69aded80450/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/tinyxml2/6.2.0-a0ad3950415fa3138d99b7da42eb4c9f/include -O3 -pthread -pipe -Werror=main -Werror=pointer-arith -Werror=overlength-strings -Wno-vla -Werror=overflow -std=c++20 -ftree-vectorize -Werror=array-bounds -Werror=format-contains-nul -Werror=type-limits -fvisibility-inlines-hidden -fno-math-errno --param vect-max-version-for-alias-checks=50 -Xassembler --compress-debug-sections -Wno-error=array-bounds -Warray-bounds -fuse-ld=bfd -march=x86-64-v3 -felide-constructors -fmessage-length=0 -Wall -Wno-non-template-friend -Wno-long-long -Wreturn-type -Wextra -Wpessimizing-move -Wclass-memaccess -Wno-cast-function-type -Wno-unused-but-set-parameter -Wno-ignored-qualifiers -Wno-unused-parameter -Wunused -Wparentheses -Werror=return-type -Werror=missing-braces -Werror=unused-value -Werror=unused-label -Werror=address -Werror=format -Werror=sign-compare -Werror=write-strings -Werror=delete-non-virtual-dtor -Werror=strict-aliasing -Werror=narrowing -Werror=unused-but-set-variable -Werror=reorder -Werror=unused-variable -Werror=conversion-null -Werror=return-local-addr -Wnon-virtual-dtor -Werror=switch -fdiagnostics-show-option -Wno-unused-local-typedefs -Wno-attributes -Wno-psabi -DEIGEN_DONT_PARALLELIZE -DEIGEN_MAX_ALIGN_BYTES=64 -Wno-error=unused-variable -ftls-model=global-dynamic -pthread -DALPAKA_DEFAULT_HOST_MEMORY_ALIGNMENT=128 -DALPAKA_DISABLE_VENDOR_RNG -DBOOST_DISABLE_ASSERTS -flto=auto -fipa-icf -flto-odr-type-merging -fno-fat-lto-objects -Wodr -fPIC -MMD -MF tmp/el8_amd64_gcc12/src/RecoEgamma/EgammaElectronProducers/plugins/RecoEgammaEgammaElectronProducersPlugins/LowPtGsfElectronSCProducer.cc.d src/RecoEgamma/EgammaElectronProducers/plugins/LowPtGsfElectronSCProducer.cc -o tmp/el8_amd64_gcc12/src/RecoEgamma/EgammaElectronProducers/plugins/RecoEgammaEgammaElectronProducersPlugins/LowPtGsfElectronSCProducer.cc.o
>> Compiling edm plugin src/RecoEgamma/EgammaElectronProducers/plugins/LowPtGsfElectronSeedHeavyObjectCache.cc
/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/gcc/12.3.1-40d504be6370b5a30e3947a6e575ca28/bin/c++ -c -DCMS_MICRO_ARCH='x86-64-v3' -DGNU_GCC -D_GNU_SOURCE -DTBB_USE_GLIBCXX_VERSION=120301 -DTBB_SUPPRESS_DEPRECATED_MESSAGES -DTBB_PREVIEW_RESUMABLE_TASKS=1 -DTBB_PREVIEW_TASK_GROUP_EXTENSIONS=1 -DBOOST_SPIRIT_THREADSAFE -DPHOENIX_THREADSAFE -DBOOST_MATH_DISABLE_STD_FPCLASSIFY -DBOOST_UUID_RANDOM_PROVIDER_FORCE_POSIX -DBOOST_MPL_IGNORE_PARENTHESES_WARNING -DGNU_GCC -DG4V9 -DGNU_GCC -DG4V9 -DDD4HEP_USE_GEANT4_UNITS=1 -DCMSSW_GIT_HASH='CMSSW_15_1_X_2025-09-01-1100' -DPROJECT_NAME='CMSSW' -DPROJECT_VERSION='CMSSW_15_1_X_2025-09-01-1100' -Isrc -Ipoison -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/cms/cmssw-patch/CMSSW_15_1_X_2025-09-01-1100/src -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/cms/coral/CORAL_2_3_21-5febecd7b3082c7b62e92719f54f5ed2/include/LCG -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/dd4hep/v01-29-00-20154c7d2051779000c69dd6f1e43dab/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/alpaka/1.2.0-ab54b9441c1c5ea12d3b50618a416376/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/geant4/11.2.2-7ace3faa1f613dee2af01c298c5e704c/include/Geant4 -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/geant4/11.2.2-7ace3faa1f613dee2af01c298c5e704c/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/libpng/1.6.37-04d5aac74fb0e0b044d856194f5a4a98/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/pcre/8.43-2d141998cfe5424b8f7aff48035cc2da/include -isystem/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/boost/1.80.0-9fe8bbbc2e1ad17df3cd25532722ba62/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/bz2lib/1.0.6-d065ccd79984efc6d4660f410e4c81de/include -isystem/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/clhep/2.4.7.1-d3a3e353d370e701238f7949a0d7909f/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/curl/7.79.0-d8c0d5017cfad573d276c56bd08114b8/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/expat/2.4.8-b093687a482bf386f8f8c236c5b2efa2/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/giflib/5.2.1-c9b71bd305c04482627fc5451da9661d/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/gsl/2.6-f7574c606b0ce57ff601d3ca9534cd01/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/heppdt/3.04.01-ed47955de25516cfc664d37543c8fc1d/include -isystem/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/hls/2025.05-4da036171b10090d7d21e15c49bb27a1/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/libjpeg-turbo/2.0.2-a09ac163eb10711ddf15e845fb1234c1/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/libuuid/2.34-27ce4c3579b5b1de2808ea9c4cd8ed29/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/protobuf/3.21.9-1126508a53768c90e66f6bf1821ac03a/include -isystem/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/lcg/root/6.32.13-be42ecc3174fc32011283ebcc8be2a76/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/sqlite/3.36.0-6a61dda6b4338fb9b6c26060efd5fd9a/include -isystem/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/tbb/v2022.0.0-d2baf29793e68d7b624be49d552519ec/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/cms/vdt/0.4.3-f008bd09702a81e8a062bbef698ed3af/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/vecgeom/v1.2.11-db93479c7d2200c4de9dccc28799f670/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/vecgeom/v1.2.11-db93479c7d2200c4de9dccc28799f670/include/VecGeom -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/xerces-c/3.1.3-c7b88eaa36d0408120f3c29826a04bf6/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/xz/5.2.5-6f3f49b07db84e10c9be594a1176c114/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/zlib/1.2.13-d217cdbdd8d586e845e05946de2796be/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/cuda/12.9.0-cd4613335deff6e5576167f7dbef4cb0/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/eigen/3bb6a48d8c171cf20b5f8e48bfb4e424fbd4f79e-5d91c922e771c0dc4f6bc00f61f3e2c5/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/eigen/3bb6a48d8c171cf20b5f8e48bfb4e424fbd4f79e-5d91c922e771c0dc4f6bc00f61f3e2c5/include/eigen3 -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/fmt/10.2.1-e35fd1db5eb3abc8ac0452e8ee427196/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/json/3.11.3-fe8b14080d05d6ff46ded7c91e3977bc/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/md5/1.0.0-5b594b264e04ae51e893b1d69a797ec6/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/OpenBLAS/0.3.27-70a9dd2c9f309171934f13e3003b0540/include -isystem/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/tensorflow/2.12.0-d8b7f62cac965299506ba69aded80450/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/tinyxml2/6.2.0-a0ad3950415fa3138d99b7da42eb4c9f/include -O3 -pthread -pipe -Werror=main -Werror=pointer-arith -Werror=overlength-strings -Wno-vla -Werror=overflow -std=c++20 -ftree-vectorize -Werror=array-bounds -Werror=format-contains-nul -Werror=type-limits -fvisibility-inlines-hidden -fno-math-errno --param vect-max-version-for-alias-checks=50 -Xassembler --compress-debug-sections -Wno-error=array-bounds -Warray-bounds -fuse-ld=bfd -march=x86-64-v3 -felide-constructors -fmessage-length=0 -Wall -Wno-non-template-friend -Wno-long-long -Wreturn-type -Wextra -Wpessimizing-move -Wclass-memaccess -Wno-cast-function-type -Wno-unused-but-set-parameter -Wno-ignored-qualifiers -Wno-unused-parameter -Wunused -Wparentheses -Werror=return-type -Werror=missing-braces -Werror=unused-value -Werror=unused-label -Werror=address -Werror=format -Werror=sign-compare -Werror=write-strings -Werror=delete-non-virtual-dtor -Werror=strict-aliasing -Werror=narrowing -Werror=unused-but-set-variable -Werror=reorder -Werror=unused-variable -Werror=conversion-null -Werror=return-local-addr -Wnon-virtual-dtor -Werror=switch -fdiagnostics-show-option -Wno-unused-local-typedefs -Wno-attributes -Wno-psabi -DEIGEN_DONT_PARALLELIZE -DEIGEN_MAX_ALIGN_BYTES=64 -Wno-error=unused-variable -ftls-model=global-dynamic -pthread -DALPAKA_DEFAULT_HOST_MEMORY_ALIGNMENT=128 -DALPAKA_DISABLE_VENDOR_RNG -DBOOST_DISABLE_ASSERTS -flto=auto -fipa-icf -flto-odr-type-merging -fno-fat-lto-objects -Wodr -fPIC -MMD -MF tmp/el8_amd64_gcc12/src/RecoEgamma/EgammaElectronProducers/plugins/RecoEgammaEgammaElectronProducersPlugins/LowPtGsfElectronSeedHeavyObjectCache.cc.d src/RecoEgamma/EgammaElectronProducers/plugins/LowPtGsfElectronSeedHeavyObjectCache.cc -o tmp/el8_amd64_gcc12/src/RecoEgamma/EgammaElectronProducers/plugins/RecoEgammaEgammaElectronProducersPlugins/LowPtGsfElectronSeedHeavyObjectCache.cc.o
In file included from src/RecoLocalCalo/HGCalRecAlgos/interface/ClusterTools.h:12,
                 from src/RecoEgamma/EgammaElectronProducers/plugins/ElectronSeedProducer.cc:30:
poison/CommonTools/RecoAlgos/interface/MultiVectorManager.h:1:2: error: #error THIS FILE HAS BEEN REMOVED FROM THE PACKAGE.
    1 | #error THIS FILE HAS BEEN REMOVED FROM THE PACKAGE.
      |  ^~~~~
>> Compiling edm plugin src/RecoEgamma/EgammaElectronProducers/plugins/LowPtGsfElectronSeedProducer.cc
/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/gcc/12.3.1-40d504be6370b5a30e3947a6e575ca28/bin/c++ -c -DCMS_MICRO_ARCH='x86-64-v3' -DGNU_GCC -D_GNU_SOURCE -DTBB_USE_GLIBCXX_VERSION=120301 -DTBB_SUPPRESS_DEPRECATED_MESSAGES -DTBB_PREVIEW_RESUMABLE_TASKS=1 -DTBB_PREVIEW_TASK_GROUP_EXTENSIONS=1 -DBOOST_SPIRIT_THREADSAFE -DPHOENIX_THREADSAFE -DBOOST_MATH_DISABLE_STD_FPCLASSIFY -DBOOST_UUID_RANDOM_PROVIDER_FORCE_POSIX -DBOOST_MPL_IGNORE_PARENTHESES_WARNING -DGNU_GCC -DG4V9 -DGNU_GCC -DG4V9 -DDD4HEP_USE_GEANT4_UNITS=1 -DCMSSW_GIT_HASH='CMSSW_15_1_X_2025-09-01-1100' -DPROJECT_NAME='CMSSW' -DPROJECT_VERSION='CMSSW_15_1_X_2025-09-01-1100' -Isrc -Ipoison -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/cms/cmssw-patch/CMSSW_15_1_X_2025-09-01-1100/src -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/cms/coral/CORAL_2_3_21-5febecd7b3082c7b62e92719f54f5ed2/include/LCG -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/dd4hep/v01-29-00-20154c7d2051779000c69dd6f1e43dab/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/alpaka/1.2.0-ab54b9441c1c5ea12d3b50618a416376/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/geant4/11.2.2-7ace3faa1f613dee2af01c298c5e704c/include/Geant4 -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/geant4/11.2.2-7ace3faa1f613dee2af01c298c5e704c/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/libpng/1.6.37-04d5aac74fb0e0b044d856194f5a4a98/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/pcre/8.43-2d141998cfe5424b8f7aff48035cc2da/include -isystem/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/boost/1.80.0-9fe8bbbc2e1ad17df3cd25532722ba62/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/bz2lib/1.0.6-d065ccd79984efc6d4660f410e4c81de/include -isystem/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/clhep/2.4.7.1-d3a3e353d370e701238f7949a0d7909f/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/curl/7.79.0-d8c0d5017cfad573d276c56bd08114b8/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/expat/2.4.8-b093687a482bf386f8f8c236c5b2efa2/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/giflib/5.2.1-c9b71bd305c04482627fc5451da9661d/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/gsl/2.6-f7574c606b0ce57ff601d3ca9534cd01/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/heppdt/3.04.01-ed47955de25516cfc664d37543c8fc1d/include -isystem/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/hls/2025.05-4da036171b10090d7d21e15c49bb27a1/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/libjpeg-turbo/2.0.2-a09ac163eb10711ddf15e845fb1234c1/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/libuuid/2.34-27ce4c3579b5b1de2808ea9c4cd8ed29/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/protobuf/3.21.9-1126508a53768c90e66f6bf1821ac03a/include -isystem/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/lcg/root/6.32.13-be42ecc3174fc32011283ebcc8be2a76/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/sqlite/3.36.0-6a61dda6b4338fb9b6c26060efd5fd9a/include -isystem/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/tbb/v2022.0.0-d2baf29793e68d7b624be49d552519ec/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/cms/vdt/0.4.3-f008bd09702a81e8a062bbef698ed3af/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/vecgeom/v1.2.11-db93479c7d2200c4de9dccc28799f670/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/vecgeom/v1.2.11-db93479c7d2200c4de9dccc28799f670/include/VecGeom -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/xerces-c/3.1.3-c7b88eaa36d0408120f3c29826a04bf6/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/xz/5.2.5-6f3f49b07db84e10c9be594a1176c114/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/zlib/1.2.13-d217cdbdd8d586e845e05946de2796be/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/cuda/12.9.0-cd4613335deff6e5576167f7dbef4cb0/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/eigen/3bb6a48d8c171cf20b5f8e48bfb4e424fbd4f79e-5d91c922e771c0dc4f6bc00f61f3e2c5/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/eigen/3bb6a48d8c171cf20b5f8e48bfb4e424fbd4f79e-5d91c922e771c0dc4f6bc00f61f3e2c5/include/eigen3 -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/fmt/10.2.1-e35fd1db5eb3abc8ac0452e8ee427196/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/json/3.11.3-fe8b14080d05d6ff46ded7c91e3977bc/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/md5/1.0.0-5b594b264e04ae51e893b1d69a797ec6/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/OpenBLAS/0.3.27-70a9dd2c9f309171934f13e3003b0540/include -isystem/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/tensorflow/2.12.0-d8b7f62cac965299506ba69aded80450/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02905/el8_amd64_gcc12/external/tinyxml2/6.2.0-a0ad3950415fa3138d99b7da42eb4c9f/include -O3 -pthread -pipe -Werror=main -Werror=pointer-arith -Werror=overlength-strings -Wno-vla -Werror=overflow -std=c++20 -ftree-vectorize -Werror=array-bounds -Werror=format-contains-nul -Werror=type-limits -fvisibility-inlines-hidden -fno-math-errno --param vect-max-version-for-alias-checks=50 -Xassembler --compress-debug-sections -Wno-error=array-bounds -Warray-bounds -fuse-ld=bfd -march=x86-64-v3 -felide-constructors -fmessage-length=0 -Wall -Wno-non-template-friend -Wno-long-long -Wreturn-type -Wextra -Wpessimizing-move -Wclass-memaccess -Wno-cast-function-type -Wno-unused-but-set-parameter -Wno-ignored-qualifiers -Wno-unused-parameter -Wunused -Wparentheses -Werror=return-type -Werror=missing-braces -Werror=unused-value -Werror=unused-label -Werror=address -Werror=format -Werror=sign-compare -Werror=write-strings -Werror=delete-non-virtual-dtor -Werror=strict-aliasing -Werror=narrowing -Werror=unused-but-set-variable -Werror=reorder -Werror=unused-variable -Werror=conversion-null -Werror=return-local-addr -Wnon-virtual-dtor -Werror=switch -fdiagnostics-show-option -Wno-unused-local-typedefs -Wno-attributes -Wno-psabi -DEIGEN_DONT_PARALLELIZE -DEIGEN_MAX_ALIGN_BYTES=64 -Wno-error=unused-variable -ftls-model=global-dynamic -pthread -DALPAKA_DEFAULT_HOST_MEMORY_ALIGNMENT=128 -DALPAKA_DISABLE_VENDOR_RNG -DBOOST_DISABLE_ASSERTS -flto=auto -fipa-icf -flto-odr-type-merging -fno-fat-lto-objects -Wodr -fPIC -MMD -MF tmp/el8_amd64_gcc12/src/RecoEgamma/EgammaElectronProducers/plugins/RecoEgammaEgammaElectronProducersPlugins/LowPtGsfElectronSeedProducer.cc.d src/RecoEgamma/EgammaElectronProducers/plugins/LowPtGsfElectronSeedProducer.cc -o tmp/el8_amd64_gcc12/src/RecoEgamma/EgammaElectronProducers/plugins/RecoEgammaEgammaElectronProducersPlugins/LowPtGsfElectronSeedProducer.cc.o
>> Compiling edm plugin src/RecoEgamma/EgammaElectronProducers/plugins/LowPtGsfElectronSeedValueMapsProducer.cc
</pre>

--- a/DataFormats/HGCalReco/interface/MultiVectorManager.h
+++ b/DataFormats/HGCalReco/interface/MultiVectorManager.h
@@ -1,6 +1,6 @@
 // Author: Felice Pantaleo (CERN), 2023, felice.pantaleo@cern.ch
-#ifndef MultiVectorManager_h
-#define MultiVectorManager_h
+#ifndef DataFormats_HGCalReco_MultiVectorManager_h
+#define DataFormats_HGCalReco_MultiVectorManager_h
 
 #include <vector>
 #include <cassert>

--- a/DataFormats/HGCalReco/interface/Trackster.h
+++ b/DataFormats/HGCalReco/interface/Trackster.h
@@ -9,7 +9,7 @@
 
 #include "DataFormats/Provenance/interface/ProductID.h"
 #include "DataFormats/Math/interface/Vector3D.h"
-#include "CommonTools/RecoAlgos/interface/MultiVectorManager.h"
+#include "DataFormats/HGCalReco/interface/MultiVectorManager.h"
 
 #include <Eigen/Core>
 

--- a/RecoHGCal/TICL/interface/TICLInterpretationAlgoBase.h
+++ b/RecoHGCal/TICL/interface/TICLInterpretationAlgoBase.h
@@ -22,7 +22,7 @@
 #include "DataFormats/HGCalReco/interface/Common.h"
 #include "FWCore/Framework/interface/ConsumesCollector.h"
 #include "PhysicsTools/TensorFlow/interface/TensorFlow.h"
-#include "CommonTools/RecoAlgos/interface/MultiVectorManager.h"
+#include "DataFormats/HGCalReco/interface/MultiVectorManager.h"
 
 namespace edm {
   class Event;

--- a/RecoHGCal/TICL/interface/TracksterLinkingAlgoBase.h
+++ b/RecoHGCal/TICL/interface/TracksterLinkingAlgoBase.h
@@ -19,7 +19,7 @@
 #include "DataFormats/HGCalReco/interface/Common.h"
 #include "FWCore/Framework/interface/ConsumesCollector.h"
 #include "PhysicsTools/TensorFlow/interface/TensorFlow.h"
-#include "CommonTools/RecoAlgos/interface/MultiVectorManager.h"
+#include "DataFormats/HGCalReco/interface/MultiVectorManager.h"
 #include "MagneticField/Engine/interface/MagneticField.h"
 #include "TrackingTools/GeomPropagators/interface/Propagator.h"
 #include "RecoLocalCalo/HGCalRecAlgos/interface/RecHitTools.h"

--- a/RecoHGCal/TICL/plugins/TICLCandidateProducer.cc
+++ b/RecoHGCal/TICL/plugins/TICLCandidateProducer.cc
@@ -1,7 +1,7 @@
 // Author: Felice Pantaleo, Wahid Redjeb, Aurora Perego (CERN) - felice.pantaleo@cern.ch, wahid.redjeb@cern.ch, aurora.perego@cern.ch
 // Date: 12/2023
 #include <memory>  // unique_ptr
-#include "CommonTools/RecoAlgos/interface/MultiVectorManager.h"
+#include "DataFormats/HGCalReco/interface/MultiVectorManager.h"
 #include "FWCore/Framework/interface/stream/EDProducer.h"
 #include "FWCore/ParameterSet/interface/ParameterSet.h"
 #include "FWCore/ParameterSet/interface/ConfigurationDescriptions.h"

--- a/RecoHGCal/TICL/plugins/TracksterLinksProducer.cc
+++ b/RecoHGCal/TICL/plugins/TracksterLinksProducer.cc
@@ -1,7 +1,7 @@
 // Author: Felice Pantaleo, Wahid Redjeb (CERN) - felice.pantaleo@cern.ch, wahid.redjeb@cern.ch
 // Date: 12/2023
 #include <memory>  // unique_ptr
-#include "CommonTools/RecoAlgos/interface/MultiVectorManager.h"
+#include "DataFormats/HGCalReco/interface/MultiVectorManager.h"
 #include "FWCore/Framework/interface/stream/EDProducer.h"
 #include "FWCore/ParameterSet/interface/ParameterSet.h"
 #include "FWCore/ParameterSet/interface/ConfigurationDescriptions.h"

--- a/RecoLocalCalo/HGCalRecAlgos/interface/ClusterTools.h
+++ b/RecoLocalCalo/HGCalRecAlgos/interface/ClusterTools.h
@@ -9,7 +9,7 @@
 #include "DataFormats/HGCRecHit/interface/HGCRecHitCollections.h"
 #include "DataFormats/CaloRecHit/interface/CaloCluster.h"
 #include "DataFormats/ParticleFlowReco/interface/HGCalMultiCluster.h"
-#include "CommonTools/RecoAlgos/interface/MultiVectorManager.h"
+#include "DataFormats/HGCalReco/interface/MultiVectorManager.h"
 
 #include "Geometry/CaloGeometry/interface/CaloGeometry.h"
 #include "Geometry/Records/interface/CaloGeometryRecord.h"

--- a/RecoLocalCalo/HGCalRecProducers/plugins/RecHitMapProducer.cc
+++ b/RecoLocalCalo/HGCalRecProducers/plugins/RecHitMapProducer.cc
@@ -13,7 +13,7 @@
 
 #include "DataFormats/HGCRecHit/interface/HGCRecHitCollections.h"
 #include "DataFormats/ParticleFlowReco/interface/PFRecHit.h"
-#include "CommonTools/RecoAlgos/interface/MultiVectorManager.h"
+#include "DataFormats/HGCalReco/interface/MultiVectorManager.h"
 
 class RecHitMapProducer : public edm::global::EDProducer<> {
 public:

--- a/SimCalorimetry/HGCalAssociatorProducers/plugins/AllHitToTracksterAssociatorsProducer.cc
+++ b/SimCalorimetry/HGCalAssociatorProducers/plugins/AllHitToTracksterAssociatorsProducer.cc
@@ -10,7 +10,7 @@
 #include "DataFormats/CaloRecHit/interface/CaloCluster.h"
 #include "SimDataFormats/Associations/interface/TICLAssociationMap.h"
 #include "DataFormats/HGCRecHit/interface/HGCRecHitCollections.h"
-#include "CommonTools/RecoAlgos/interface/MultiVectorManager.h"
+#include "DataFormats/HGCalReco/interface/MultiVectorManager.h"
 
 class AllHitToTracksterAssociatorsProducer : public edm::global::EDProducer<> {
 public:

--- a/SimCalorimetry/HGCalAssociatorProducers/plugins/AllTracksterToSimTracksterAssociatorsByHitsProducer.cc
+++ b/SimCalorimetry/HGCalAssociatorProducers/plugins/AllTracksterToSimTracksterAssociatorsByHitsProducer.cc
@@ -10,7 +10,7 @@
 #include "SimDataFormats/Associations/interface/TICLAssociationMap.h"
 #include "DataFormats/Provenance/interface/ProductID.h"
 #include "DataFormats/HGCRecHit/interface/HGCRecHitCollections.h"
-#include "CommonTools/RecoAlgos/interface/MultiVectorManager.h"
+#include "DataFormats/HGCalReco/interface/MultiVectorManager.h"
 #include "DataFormats/CaloRecHit/interface/CaloCluster.h"
 #include "SimDataFormats/CaloAnalysis/interface/CaloParticle.h"
 #include "SimDataFormats/CaloAnalysis/interface/SimCluster.h"

--- a/SimCalorimetry/HGCalAssociatorProducers/plugins/HitToLayerClusterAssociatorProducer.cc
+++ b/SimCalorimetry/HGCalAssociatorProducers/plugins/HitToLayerClusterAssociatorProducer.cc
@@ -11,7 +11,7 @@
 #include "DataFormats/CaloRecHit/interface/CaloCluster.h"
 #include "DataFormats/HGCRecHit/interface/HGCRecHitCollections.h"
 #include "SimDataFormats/Associations/interface/TICLAssociationMap.h"
-#include "CommonTools/RecoAlgos/interface/MultiVectorManager.h"
+#include "DataFormats/HGCalReco/interface/MultiVectorManager.h"
 
 HitToLayerClusterAssociatorProducer::HitToLayerClusterAssociatorProducer(const edm::ParameterSet &pset)
     : LCCollectionToken_(consumes<std::vector<reco::CaloCluster>>(pset.getParameter<edm::InputTag>("layer_clusters"))),

--- a/SimCalorimetry/HGCalAssociatorProducers/plugins/HitToLayerClusterAssociatorProducer.h
+++ b/SimCalorimetry/HGCalAssociatorProducers/plugins/HitToLayerClusterAssociatorProducer.h
@@ -18,7 +18,7 @@
 #include "DataFormats/CaloRecHit/interface/CaloCluster.h"
 #include "DataFormats/HGCRecHit/interface/HGCRecHitCollections.h"
 #include "SimDataFormats/Associations/interface/TICLAssociationMap.h"
-#include "CommonTools/RecoAlgos/interface/MultiVectorManager.h"
+#include "DataFormats/HGCalReco/interface/MultiVectorManager.h"
 
 class HitToLayerClusterAssociatorProducer : public edm::global::EDProducer<> {
 public:

--- a/SimCalorimetry/HGCalAssociatorProducers/plugins/HitToSimClusterCaloParticleAssociatorProducer.cc
+++ b/SimCalorimetry/HGCalAssociatorProducers/plugins/HitToSimClusterCaloParticleAssociatorProducer.cc
@@ -11,7 +11,7 @@
 #include "SimDataFormats/Associations/interface/TICLAssociationMap.h"
 #include "DataFormats/Provenance/interface/ProductID.h"
 #include "DataFormats/HGCRecHit/interface/HGCRecHitCollections.h"
-#include "CommonTools/RecoAlgos/interface/MultiVectorManager.h"
+#include "DataFormats/HGCalReco/interface/MultiVectorManager.h"
 #include "SimDataFormats/CaloAnalysis/interface/CaloParticle.h"
 #include "SimDataFormats/CaloAnalysis/interface/SimCluster.h"
 

--- a/SimCalorimetry/HGCalAssociatorProducers/plugins/HitToSimClusterCaloParticleAssociatorProducer.h
+++ b/SimCalorimetry/HGCalAssociatorProducers/plugins/HitToSimClusterCaloParticleAssociatorProducer.h
@@ -16,7 +16,7 @@
 #include "SimDataFormats/Associations/interface/TICLAssociationMap.h"
 #include "DataFormats/Provenance/interface/ProductID.h"
 #include "DataFormats/HGCRecHit/interface/HGCRecHitCollections.h"
-#include "CommonTools/RecoAlgos/interface/MultiVectorManager.h"
+#include "DataFormats/HGCalReco/interface/MultiVectorManager.h"
 #include "SimDataFormats/CaloAnalysis/interface/CaloParticle.h"
 #include "SimDataFormats/CaloAnalysis/interface/SimCluster.h"
 

--- a/SimCalorimetry/HGCalAssociatorProducers/plugins/HitToTracksterAssociatorProducer.cc
+++ b/SimCalorimetry/HGCalAssociatorProducers/plugins/HitToTracksterAssociatorProducer.cc
@@ -13,7 +13,7 @@
 #include "SimDataFormats/Associations/interface/TICLAssociationMap.h"
 #include "DataFormats/Provenance/interface/ProductID.h"
 #include "DataFormats/HGCRecHit/interface/HGCRecHitCollections.h"
-#include "CommonTools/RecoAlgos/interface/MultiVectorManager.h"
+#include "DataFormats/HGCalReco/interface/MultiVectorManager.h"
 
 HitToTracksterAssociatorProducer::HitToTracksterAssociatorProducer(const edm::ParameterSet &pset)
     : LCCollectionToken_(consumes<std::vector<reco::CaloCluster>>(pset.getParameter<edm::InputTag>("layer_clusters"))),

--- a/SimCalorimetry/HGCalAssociatorProducers/plugins/HitToTracksterAssociatorProducer.h
+++ b/SimCalorimetry/HGCalAssociatorProducers/plugins/HitToTracksterAssociatorProducer.h
@@ -15,7 +15,7 @@
 #include "DataFormats/CaloRecHit/interface/CaloCluster.h"
 #include "SimDataFormats/Associations/interface/TICLAssociationMap.h"
 #include "DataFormats/Provenance/interface/ProductID.h"
-#include "CommonTools/RecoAlgos/interface/MultiVectorManager.h"
+#include "DataFormats/HGCalReco/interface/MultiVectorManager.h"
 #include "DataFormats/HGCRecHit/interface/HGCRecHitCollections.h"
 
 class HitToTracksterAssociatorProducer : public edm::global::EDProducer<> {

--- a/Validation/HGCalValidation/interface/BarrelVHistoProducerAlgo.h
+++ b/Validation/HGCalValidation/interface/BarrelVHistoProducerAlgo.h
@@ -23,7 +23,7 @@
 #include "DataFormats/ParticleFlowReco/interface/PFRecHit.h"
 
 #include "RecoLocalCalo/HGCalRecAlgos/interface/RecHitTools.h"
-#include "CommonTools/RecoAlgos/interface/MultiVectorManager.h"
+#include "DataFormats/HGCalReco/interface/MultiVectorManager.h"
 
 #include "SimDataFormats/CaloAnalysis/interface/CaloParticle.h"
 #include "SimDataFormats/Vertex/interface/SimVertex.h"

--- a/Validation/HGCalValidation/interface/BarrelValidator.h
+++ b/Validation/HGCalValidation/interface/BarrelValidator.h
@@ -32,7 +32,7 @@
 #include "SimDataFormats/Associations/interface/LayerClusterToSimClusterAssociator.h"
 #include "SimDataFormats/Associations/interface/TICLAssociationMap.h"
 
-#include "CommonTools/RecoAlgos/interface/MultiVectorManager.h"
+#include "DataFormats/HGCalReco/interface/MultiVectorManager.h"
 
 class PileupSummaryInfo;
 

--- a/Validation/HGCalValidation/interface/HGCalValidator.h
+++ b/Validation/HGCalValidation/interface/HGCalValidator.h
@@ -32,7 +32,7 @@
 #include "SimDataFormats/Associations/interface/LayerClusterToSimClusterAssociator.h"
 #include "SimDataFormats/Associations/interface/TICLAssociationMap.h"
 
-#include "CommonTools/RecoAlgos/interface/MultiVectorManager.h"
+#include "DataFormats/HGCalReco/interface/MultiVectorManager.h"
 
 class PileupSummaryInfo;
 

--- a/Validation/HGCalValidation/interface/HGVHistoProducerAlgo.h
+++ b/Validation/HGCalValidation/interface/HGVHistoProducerAlgo.h
@@ -22,7 +22,7 @@
 #include "DataFormats/HGCalReco/interface/Trackster.h"
 
 #include "RecoLocalCalo/HGCalRecAlgos/interface/RecHitTools.h"
-#include "CommonTools/RecoAlgos/interface/MultiVectorManager.h"
+#include "DataFormats/HGCalReco/interface/MultiVectorManager.h"
 
 #include "SimDataFormats/CaloAnalysis/interface/CaloParticle.h"
 #include "SimDataFormats/Vertex/interface/SimVertex.h"

